apptuit/apptuit_client.py

```diff
@@ -69,12 +59,44 @@ def _parse_response(resp, start, end=None):
             qresult[output_id].series.append(series)
     return qresult
 
+def _get_token_from_environment():
+    token = os.environ.get(APPTUIT_API_TOKEN)
+    return token
+
+def _get_tags_from_environment():
+    tags_str = os.environ.get(APPTUIT_PY_TAGS)
+    if not tags_str:
+        tags = {}
+        return tags
+    tags = {}
+    tags_split = tags_str.split(',')
+    for tag in tags_split:
+        try:
+            key, val = tag.split(":")
```

Comment:
can we specify in the exception message that this particular tag had bad format?